.github/workflows/python-package.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pytest
+        #pip install pytest
```

Comment:
Isn't this required by the command below?

Reply:
Yeah but it's included in our requirements already.